src/System.Private.CoreLib/src/System/Threading/Thread.CoreRT.Windows.cs

```diff
@@ -126,6 +127,24 @@ private bool SetPriorityLive(ThreadPriority priority)
             return Interop.Kernel32.SetThreadPriority(_osHandle, (int)MapToOSPriority(priority));
         }
 
+        [UnmanagedCallersOnly]
```

Comment:
This is almost identical between the Windows and Unix versions of the file. It would be nice to move it to the common file.